packages/boot/tools/supports.ts

```diff
@@ -171,6 +173,8 @@ export const getNodeTestVaultsConfig = async ({
     );
   }
 
+  Object.assign(config, configOverrides);
```

Comment:
This relies on `loadSwingsetConfigFile` not hardening its result, right? That seems odd.